server/aws-lsp-partiql/package.json

```diff
@@ -13,15 +13,19 @@
     "scripts": {
         "compile": "tsc --build",
         "test": "jest",
+        "precompile": "npm run update-treesitter-wasm",
```

Comment:
The `precompile` script is run every time `compile` is run, which can be quite often during development. Do we need to update the treesitter wasm every time wel compile? I think it only needs to be updated sometimes right?

Reply:
Yea, this script is only needed to and has to run whenever the package version `web-tree-sitter` upgrades. To make sure the script runs when needed, I will force the `web-tree-sitter` version to be "0.22.6" for now and leave an update instruction message in README.md.